### ISSUE TITLE
Display ability/feat notes at the end of the statblock

### ISSUE THREAD

**Issue by niertis** [Mottokrosh/SheetIssues#14]
In a future enhancement it would be nice to view the notes section from the special abilities and feats section at the bottom of the statblock.

Such as the image below:
![image](https://cloud.githubusercontent.com/assets/4295793/3696947/2d47e530-139a-11e4-8296-fa61da77a524.png)

**Comment by MythicParty** [comment]
Agreed, but I'd suggest being able to toggle on/off each ability so when you could better control the amount of text both on the screen & that gets printed.  i.e. often not every ability needs to be out there
